spec/sidekiq-instrument/client_middleware_spec.rb

```diff
@@ -20,50 +30,34 @@
 
     context 'without statsd_metric_name' do
       it 'increments the StatsD enqueue counter' do
-        expect {
+        expect do
           MyWorker.perform_async
-        }.to trigger_statsd_increment('shared.sidekiq.default.MyWorker.enqueue')
+        end.to trigger_statsd_increment('shared.sidekiq.default.MyWorker.enqueue')
       end
 
       it 'increments the DogStatsD enqueue counter' do
-        expect(Sidekiq::Instrument::Statter.dogstatsd).to receive(:increment).with('sidekiq.enqueue', { tags: ['queue:default', 'worker:my_worker'] }).once
+        expect(
+          Sidekiq::Instrument::Statter.dogstatsd
+          ).to receive(:increment).with('sidekiq.enqueue', { tags: ['queue:default', 'worker:my_worker'] }).once
         MyWorker.perform_async
       end
     end
 
     context 'with statsd_metric_name' do
       it 'increments the enqueue counter' do
-        expect {
+        expect do
           MyOtherWorker.perform_async
-        }.to trigger_statsd_increment('my_other_worker.enqueue')
+        end.to trigger_statsd_increment('my_other_worker.enqueue')
       end
     end
 
     context 'with WorkerMetrics.enabled true' do
-      let(:worker_metric_name) do
-        "sidekiq_instrument_trace_workers::in_queue"
-      end
       it 'increments the enqueue counter' do
```

Comment:
Should this be `in_queue counter` (or `in queue counter` to match the verbiage chosen in other [new](https://github.com/enova/sidekiq-instrument/pull/24/files#diff-bf3ae6f4843f969152133a71a1b11eeda8c87da740d816f88e1a9f4220fae5d4R54) [tests](https://github.com/enova/sidekiq-instrument/pull/24/files#diff-bf3ae6f4843f969152133a71a1b11eeda8c87da740d816f88e1a9f4220fae5d4R63))? (since we're exercising the `WorkerMetrics` and observing the `worker_metric_name` (which is `sidekiq_instrument_trace_workers::in_queue`))
```suggestion
      it 'increments the in queue counter' do
```